co.uk.studiopond.spiderbro.yml

```diff
@@ -0,0 +1,44 @@
+app-id: co.uk.studiopond.spiderbro
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: spiderbro_linux
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+modules:
+  - shared-modules/glu/glu-9.json
+  - name: Spiderbro
+    buildsystem: simple
+    build-commands:
+      - install -D spiderbro_linux /app/bin/spiderbro_linux
+      - install -D polybius_linux /app/bin/polybius_linux
+      - unzip -o data.zip -d /app/bin
+      - unzip -o polydata.zip -d /app/bin
+      - install -D co.uk.studiopond.spiderbro.svg /app/share/icons/hicolor/scalable/apps/co.uk.studiopond.spiderbro.svg
+      - install -D co.uk.studiopond.spiderbro.desktop /app/share/applications/co.uk.studiopond.spiderbro.desktop
+    sources:
+      - type: file
+        url: https://github.com/pforpond/Spiderbro/blob/b3eb380755d6e4b79e13dd86d28e65e73470b7a9/flatpak/spiderbro_linux?raw=true
+        sha256: e831fb578ceefe8c17956cb27c4bf35ca4ed946df133e4796862d75876ff3a80
+        only-arches: ["x86_64"]
+      - type: file
+        url: https://github.com/pforpond/Spiderbro/blob/main/flatpak/data.zip?raw=true
+        sha256: 0ea9ce61d1e537a1d7146da992917113e77f3fc94051592a3c79f4012faf9b0f
+        only-arches: ["x86_64"]
+      - type: file
+        url: https://github.com/pforpond/Spiderbro/blob/main/flatpak/polybius_linux?raw=true
+        sha256: 36e09a1b73216cb4a176d8721e4c816f4c55e8d7b6f6cb854151c1677b574c9d
+        only-arches: ["x86_64"]
+      - type: file
+        url: https://github.com/pforpond/Spiderbro/blob/main/flatpak/polydata.zip?raw=true
+        sha256: 8d02a9611434bf50ac1803dc6bd4d4547437c6c16e5ca58ca3ed41525e969f5f
+        only-arches: ["x86_64"]
+      - type: file
+        url: https://github.com/pforpond/Spiderbro/blob/main/flatpak/co.uk.studiopond.spiderbro.svg?raw=true
+        sha256: 669d9c38ba6494765834fd98852d1d795a36923b711ddc527e6376c0ad60fa69
+      - type: file
+        url: https://github.com/pforpond/Spiderbro/blob/main/flatpak/co.uk.studiopond.spiderbro.desktop?raw=true
```

Comment:
you should NEVER reference a file from a branch in git. This apply for the other above.